File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        temp = ""
        for char in s:
            if char.isalnum():
                temp += char.lower()
        l = len(temp)
        for i in range(0,l/2):
            if temp[i] != temp[l-i-1]:
                return False
        return True 

        

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string temp = "";
        for(int i = 0; i < s.size(); i++){
            if(isalnum(s[i])){
                temp += tolower(s[i]);
            }
        }
        int l = temp.size();
        for(int i = 0; i < l/2; i++){
            if(temp[i] != temp[l-i-1]) return false;
        }
        return true;
    }
};